NGitLab/IUserClient.cs

```diff
@@ -57,5 +58,21 @@ public interface IUserClient
         /// Get a list of users that match the query.
         /// </summary>
         IEnumerable<User> Get(UserQuery query);
+
+        /// <summary>
+        /// Gets the last activity date for all users, sorted from oldest to newest
+        /// </summary>
+        /// <param name="from">Date from which activities will be considered. If unspecified, will look back over the last 6 months.</param>
+        /// <remarks>
+        /// The activities that update the timestamp are:
+        /// <list type="bullet">
+        /// <item><description>Git HTTP/SSH activities (such as clone, push)</description></item>
+        /// <item><description>User logging in to GitLab</description></item>
+        /// <item><description>User visiting pages related to dashboards, projects, issues, and merge requests (introduced in GitLab 11.8)</description></item>
+        /// <item><description>User using the API</description></item>
+        /// <item><description>User using the GraphQL API</description></item>
+        /// </list>
+        /// </remarks>
+        IEnumerable<LastActivityDate> GetLastActivityDates(DateTimeOffset? from = null);
```

Comment:
New methods should provide async calls. `GitLabCollectionResponse` implements `IEnumerable` and `IAsyncEnumerable`, so it is the way to go for collections. Also, you need to add the `Async` suffix for consistency (even if it could be synchronous)

```suggestion
        GitLabCollectionResponse<LastActivityDate> GetLastActivityDatesAsync(DateTimeOffset? from = null);
```